src/dmd/intrange.d

```diff
@@ -43,7 +41,7 @@ struct SignExtendedNumber
 
     static SignExtendedNumber max()
     {
-        return SignExtendedNumber(UINT64_MAX, false);
```

Comment:
Why is this refactoring here? Perhaps make it a separate PR (as this one is very large)?